release-team/role-handbooks/docs/how-tos/Take-over-a-doc-pr.md

```diff
@@ -0,0 +1,49 @@
+# Taking over a Doc PR
```

Comment:
Hey @Princesso!
I know you requested some extra details on how I went about this. Let me know if this needs any additional clarifications. Thanks!